index.html

```diff
@@ -0,0 +1,63 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Portfolio Hana Sabih</title>
+    <link rel="stylesheet" href="style.css" />
+    <script src="https://code.iconify.design/iconify-icon/1.0.2/iconify-icon.min.js"></script>
+    <!-- <link href="https://fonts.googleapis.com/css?family=Material+Icons|Material+Icons+Outlined|Material+Icons+Two+Tone|Material+Icons+Round|Material+Icons+Sharp" rel="stylesheet"> -->
+  </head>
+  <body>
+    <header>
+      <h5>Welcome</h5>
+      <iconify-icon
+        icon="ic:round-menu"
+        width="24"
+        height="24"
+        class="menubar"
+      ></iconify-icon>
+    </header>
+    <section class="headline">
+      <h1>
+        Hey there. I'm Mariane<br />
+        <span> I'm a software developer</span>
+      </h1>
+      <!-- <h3>Software Developer</h3> -->
+
+      <p>
+        I can help you build a product , feature or website Look through some of
+        my work and experience! If you like what you see and have a project you
+        need coded, don’t hestiate to contact me.
+      </p>
+      <div class="icons-contact">
+        <iconify-icon
+          icon="uim:github-alt"
+          width="20"
+          height="20"
+          class="social-icons"
+        ></iconify-icon>
+        <iconify-icon
+          icon="ri:linkedin-fill"
+          width="20"
+          height="20"
+          class="social-icons"
+        ></iconify-icon>
+        <iconify-icon
+          icon="mdi:twitter"
+          width="20"
+          height="20"
+          class="social-icons"
+        ></iconify-icon>
+
+        <iconify-icon
+          icon="fa-brands:medium-m"
+          width="20"
```

Comment:
- [ ] You should stick to the Figma design as much as possible to make your work looks more professional, so you can try to leverage the sections of icons so that it matches with Figma design in terms of spacing, colors, size, etc.

Figma design:
![image](https://user-images.githubusercontent.com/90524466/205323291-42f7a3d9-b724-4140-b7b9-41c389902b56.png)

Your design:
![image](https://user-images.githubusercontent.com/90524466/205323380-dc6d0fcf-5a9f-41d7-a9bd-133eca51a456.png)